src/SwipeButton.tsx

```diff
@@ -63,140 +77,177 @@ export type SwipeButtonCommonProps = {
 
   /**
    * The border radius of the container and the Icon
-   * 
-   * @default (height / 2)
+   *
+   * @default 100
    */
   borderRadius?: number;
-}
+};
 
-export const DEFAULT_HEIGHT = 70;
-const DEFAULT_WIDTH = Dimensions.get('window').width * 0.9;
-const DEFAULT_BORDER_RRADIUS = DEFAULT_HEIGHT / 2;
-const DEFAULT_COMPLETE_THRESHOLD_PERCENTAGE = 70;
+const DEFAULT_HEIGHT = 70;
+const DEFAULT_BORDER_RADIUS = 100;
+const DEFAULT_COMPLETE_THRESHOLD_PERCENTAGE = 100;
 
 const SwipeButton = ({
-    height = DEFAULT_HEIGHT, width = DEFAULT_WIDTH, borderRadius = DEFAULT_BORDER_RRADIUS,
-    title, titleContainerProps, titleProps, titleContainerStyle, titleStyle,
-    completeThresholdPercentage = DEFAULT_COMPLETE_THRESHOLD_PERCENTAGE,
-    underlayStyle, disabled, Icon, containerStyle,
-    onComplete, onSwipeEnd = () => {}, onSwipeStart = () => {},
+  height = DEFAULT_HEIGHT,
+  borderRadius = DEFAULT_BORDER_RADIUS,
+  title,
+  titleContainerProps,
+  titleProps,
+  titleContainerStyle,
+  titleStyle,
+  completeThresholdPercentage = DEFAULT_COMPLETE_THRESHOLD_PERCENTAGE,
+  underlayStyle,
+  disabled,
+  Icon,
+  iconContainerStyle,
+  containerStyle,
+  onComplete,
+  onSwipeEnd = () => {},
+  onSwipeStart = () => {},
```

Comment:
Thanks. This is much more readable 💪

Reply:
Ah not a problem 😄 I think my linter actually made more changes to the code than I did myself so it seems like a lot 😛